src/test/java/emissary/config/ServiceConfigGuideTest.java

```diff
@@ -513,7 +510,7 @@ void testMergeRemoval() throws IOException {
     @Test
     void testImportFileWhenFileExists() throws IOException {
```

Comment:
these could be updated to :

`void testImportFileWhenFileExists(@TempDir Path dir) throws IOException {`

and then remove the finally

Reply:
This is a systemic update that needs to be done in other places too. I'm good with pushing that to a different update.